README.md

```diff
@@ -447,6 +447,17 @@ gb.push()
 gb.start(10.0)
 ```
 
+Graphite [tags](https://grafana.com/blog/2018/01/11/graphite-1.1-teaching-an-old-dog-new-tricks/) are also supported.
+
+```python
+from prometheus_client.bridge.graphite import GraphiteBridge
+
+gb = GraphiteBridge(('graphite.your.org', 2003, tags=True))
```

Comment:
Too many things in the tuple here :)

It should be `GraphiteBridge(('graphite.your.org', 2003), tags=True)`